docs/guide-tequila-api-key.md

```diff
@@ -1,6 +1,20 @@
 ---
-title: "Guide: Get your Tequila API key"
-sidebar_label: "Guide: Get your Tequila API key"
+title: Guide: Get your Tequila API key
+sidebar_label: Guide: Get your Tequila API key
 ---
 
-_This page is a stub._
+For successful run our the GraphQL proxy server, you need to has set the Tequila **API key** to `API_KEY` environment variable.
+
+To get your **API key** you will need an account at the [Tequila](tequila.kiwi.com)
+. You can register and log in on the tequila.kiwi.com page.
+
+After successfully signing in to your account, you can find out what **Tequila** is about, how to work directly with the **Tequila API**.
+
+To get your **API key** for the GraphQL got into the My applications page.
+Create the new app, by clicking on the `add application` button.
```

Comment:
To get your **API key** go to the My Applications page and create the new app, by clicking on the `Add Application` button.

> go to the My Applications page

Can the "My Application page" be a link?

Reply:
done